cwms-data-api/src/main/java/cwms/cda/data/dto/timeseriesprofile/TimeSeriesProfileParserIndexed.java

```diff
@@ -21,6 +21,8 @@
 @JsonDeserialize(builder = TimeSeriesProfileParserIndexed.Builder.class)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+// Uses custom serializer to handle polymorphic serialization
```

Comment:
use javadocs as they can `@link` to other classes

Reply:
Moved to javadocs, now links to base class